README.md

```diff
@@ -13,38 +14,117 @@ At this time there are no web server-side components.
 
 ## DaoStack-Arc-Js
 
-Alchemy uses a library of reusable and configurable Ethereum smart contracts called "DaoStack-Arc-Js".  You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).
+Alchemy uses a library of reusable and configurable Ethereum smart contracts called "DaoStack-Arc-Js". You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).
```

Comment:
> [here](https://github.com/daostack/arc-js).

This is not the contracts repo.

Reply:
Some of the references to arc-js will have to wait until I upgrade Vanille to work with a newer (and refactored) package of arc.js.  That needs to be in another PR.  (see https://github.com/daostack/Vanille/issues/25)

Other references to "arc-js" will have to remain in any case, namely in the names of the scripts -- the dot is a special character in those names.

Reply:
You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).

Contracts are in Arc (not arc-js).

Reply:
OK, see what you think...